addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -959,13 +959,16 @@ body.editor_enable.editor_has_snippets {
                 }
             }
             .o_we_pager_next, .o_we_pager_prev {
+                margin: 3px;
                 padding: $o-we-sidebar-content-field-label-spacing;
                 cursor: pointer;
+                border: 1px solid;
+                border-radius: 2px;
             }
             we-select-page {
                 display: none;
                 width: 100%;
-                max-height: 600px;
+                max-height: 562.5px;
```

Comment:
562.5px? :smile: I feel like it is some formula with 9/16 you are using but it should be written that way then.

Can you move this in a PR for 14.0 instead? I feel like that menu should be improved there instead of master. Also, maybe you could make it so clicking on the shape dropdown does not close it if you misclick next to a pager button?

Reply:
Actually the <we-select-page> has a max-height of 600px; which shows 8 items (.o_we_shape_btn_content) of height:75px.
To crop "1/2" from last item, the new max-height should be : 600 - 75/2  = 562.5px :)